src/Controls/src/Core/Compatibility/Handlers/Shell/iOS/ShellFlyoutContentRenderer.cs

```diff
@@ -261,7 +261,7 @@ void UpdateFlyoutBgImageAsync()
 				return;
 			}
 
-			imageSource.LoadImage(imageSource.FindMauiContext(), result =>
+			imageSource.LoadImage(imageSource.FindMauiContext(true), result =>
```

Comment:
I think we can use _shellContext.Shell.FindMauiContext() here and that'll work

Reply:
I thought that too, but I wasn't sure what was the best context to use there

Reply:
yea the Image that's being pulled here is a property on shell so it's at the right level.